lib/grpc/auth_context/auth_context.go

```diff
@@ -78,6 +78,18 @@ func NewOutgoingProjectsContext(ctx context.Context) context.Context {
 	return metadata.NewOutgoingContext(ctx, md)
 }
 
+// ContextWithoutProjects removes previously added projects from the GRPC metadata
+// for those system operations that must not be filtered by projects.
+func ContextWithoutProjects(ctx context.Context) context.Context {
```

Comment:
One relevant detail is missing from comment and name: the translation from incoming to outgoing metadata. Can we fix this? 😉

Reply:
Sure... but I am not sure on what is what; please suggest how to make it better!

Reply:
See above, I wonder if this helper is a good idea as is. The only point where we need this incoming/outgoing translation is the gateway.

Reply:
Well, it's got a use for sure. Just not inside authz-service. 😉